Condor time

kreczko opened the pull request:
To make people more aware of this.
Needs to be tidied up before any merge attempt:
 - [x] provide job mapper (condor process id -> payload)
   - [x] unit tests for job mapper
 - [x] provide condor script for submission
   - [x] add ```--noop``` option for test purpose (would only create the job description)
 - [x] documentation
 - [x] ```--test``` argument doesn't set any default options
 - [x] help message in job_mapper is wrong

kreczko commented:
@jjacob @EmyrClement 
Ready for review/mergin

jjacob commented:
Just testing this for submitting the final analysis jobs. Currently:
- [ ] ```--test``` argument doesn't set any default options
- [ ] help message in job_mapper is wrong

jjacob commented:
OK, think this is ok to merge. I have some further changes to the job_mapper etc. to make which I will make a pull request with once this is merged.

kreczko commented:
I still need to update with datasets (i.e. import them from the master configs).

kreczko commented:
@jjacob : I have fixed the problem with the datasets and added an operation to allow submission of a single sample for all analysis modes (single-sample-analysis)
